Userland/DevTools/HackStudio/Language.cpp

```diff
@@ -19,6 +18,8 @@ Language language_from_file(LexicalPath const& file)
     if (extension == "c" || extension == "cc" || extension == "cxx" || extension == "cpp" || extension == "c++"
         || extension == "h" || extension == "hh" || extension == "hxx" || extension == "hpp" || extension == "h++")
         return Language::Cpp;
+    if (extension == "cmake" || (extension == "txt" && file.title() == "CMakeLists"))
```

Comment:
Yikes, this is three times we repeat all this same info! :( Now I'm really leaning into wanting an enhancement issue to make this filename-->language information more ergonomic and centralized.

Reply:
I know right! I was going to merge them in this PR but it's not entirely straightforward so I put it on the yak stack for later.